src/App.js

```diff
@@ -14,11 +16,120 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: [' Jam'],
```

Comment:
```suggestion
    selectedGoods: ['Jam'],
```
их же там может быть много